mtp_cashbook/apps/mtp_auth/forms.py

```diff
@@ -7,31 +8,63 @@
 from mtp_common.user_admin.forms import SignUpForm
 from oauthlib.oauth2 import OAuth2Error
 from requests import RequestException
+from zendesk_tickets.forms import BaseTicketForm
 
 logger = logging.getLogger('mtp')
 
 ACCOUNT_REQUEST_ROLE = 'prison-clerk'
 
 
-class CashbookSignUpForm(SignUpForm):
+class CashbookSignUpForm(SignUpForm, BaseTicketForm):
     prison = forms.ChoiceField(label=_('Prison'), help_text=_('Enter the prison you’re based in'))
 
+    # Non form class variables
+    cashbook_account_request_zendesk_subject = 'Requesting a new staff account - Digital Cashbook'
+    zendesk_tags = ('mtp', 'cashbook', 'account_request')
```

Comment:
could we also add `settings.ENVIRONMENT` like in https://github.com/ministryofjustice/money-to-prisoners-cashbook/blob/b62fcd3922481b651c3d5e303a8b20efc3dc6057/mtp_cashbook/apps/feedback/views.py#L13 so that hosting environments can be distinguished?

Reply:
i'm unsure about the 'feedback' tag cos it's on all our tickets currently… which means it's actually not useful for grouping tickets in any way.

Reply:
I agree with your point about the feedback tag, i'll add here anyway for consistency